test/test_c10d.py

```diff
@@ -2967,6 +2967,8 @@ def _test_broadcast_coalesced(self, process_group, device):
     def _run_all_reduce(self, pg):
         pg.allreduce(torch.rand(10).cuda(self.rank))
 
+    # Need 2.4+ for error checking.
+    @requires_nccl_version(2400)
     @requires_nccl()
```

Comment:
`requires_nccl` needs to be before `requires_nccl_version`? Or we can `c10d.is_nccl_available()` to `requires_nccl_version` as well.